tests/conftest.py

```diff
@@ -59,6 +59,24 @@ def admin(db):
     db.add(user)
     db.commit()
     db.refresh(user)
+
+    return user
+
+
+@pytest.fixture(scope="function")
+def annotator(db):
```

Comment:
We already have `AnnotatorFactory` instead of using this fixture you can do:

```python
annotator = AnnotatorFactory.create()
```

And use `annotator` where necessary.